compiler/src/org.graalvm.compiler.lir/src/org/graalvm/compiler/lir/gen/LIRGeneratorTool.java

```diff
@@ -268,6 +268,26 @@ default Variable emitStringIndexOf(Value sourcePointer, Value sourceCount, Value
         throw GraalError.unimplemented("String.indexOf substitution is not implemented on this architecture");
     }
 
+    // The routines emitStringLatin1Inflate/3 and emitStringUTF16Compress/3 models a simplified
```

Comment:
Please use `/* ... */` style comments for multiline comments. If you want to prevent `mx eclipseformat` from re-formatting the comment, use `// @formatter:off`  and `// @formatter:on`. For example:
```
    // @formatter:off
    /*
     * The routines emitStringLatin1Inflate/3 and emitStringUTF16Compress/3
     * models a simplified version of:
     *
     *    emitStringLatin1Inflate(Value src, Value src_ndx, Value dst, Value dst_ndx, Value len)
     *
     * and
     *
     *    emitStringUTF16Cpmpress(Value src, Value src_ndx, Value dst, Value dst_ndx, Value len)
     *
     * respectively, where we have hoisted the offset address computations
     * in a method replacement snippet.
     */
    // @formatter:on
```